coop_account_check_deposit/models/account_check_deposit.py

```diff
@@ -76,19 +76,23 @@ def validate_deposit(self):
                 to_reconcile_lines.append(line + move_line)
 
             # Create counter-part
-            if not deposit.destination_journal_id.default_debit_account_id:
+            if (
+                    deposit.destination_journal_id and
+                    not deposit.destination_journal_id
+                    .default_debit_account_id):
```

Comment:
You can leave it how it was. An empty recordset will just return False on its fields
So this works:

```suggestion
    if not deposit.destination_journal_id.default_debit_account_id:
```

Reply:
but in this case it raise error "Default Debit Account is not set on journal".
because account_check_deposit test case has no destination_journal_id

Reply:
Ah! I understand